tests/manage/pv_services/conftest.py

```diff
@@ -139,7 +154,7 @@ def factory(
         # pods_dc will be an empty list if deployment_config is False
         for pod_dc in pods_dc:
             pod_objs = pod.get_all_pods(
-                namespace=project.namespace,
+                namespace=pvcs[0].project.namespace,
```

Comment:
Should we check if `pvcs` is an empty list or assume that it always has at least one element?

Reply:
```pvcs``` will contain at least one element. Only possibility of empty ```pvcs``` list is if the value of both ```num_of_cephfs_pvc``` and ```num_of_rbd_pvc``` are passed as  0 which is not a correct use case.

Reply:
Yes, I see. It makes sense.